README.md

```diff
@@ -99,6 +99,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [threeschwifty](https://ngokevin.github.com/threeschwifty) &mdash; [Code](https://github.com/ngokevin/threeschwifty) - 360-degree photo and video viewer.
 - [VR GIF](http://x627.com/vr-gif/) &mdash; [Post](http://x627.com/a-frame-is-awesome/) - GIFs in VR.
 - [Virtual Art School Tour](http://stampswebdesign.com/withersc/hud/index.html) (by [@withersc]) - 360-degree photos with navigation.
+- [360-Degree Photosphere Viewer Example](https://github.com/IdeaSpaceVR/aframe-360-degree-photosphere) - simple 360-degree viewer for equirectangular photos with a navigation.
```

Comment:
Can you link to [your example](https://www.ideaspacevr.org/examples/aframe/aframe-360-degree-photosphere/) and then link to the code by doing `&mdash; [Code](https://github.com/IdeaSpaceVR/aframe-360-degree-photosphere)` (like [this one](https://github.com/aframevr/awesome-aframe/pull/25/files#diff-04c6e90faac2675aa89e2176d2eec7d8R9) above)?